pkg/vfs/vfs_unix.go

```diff
@@ -290,3 +291,27 @@ func (v *VFS) Flock(ctx Context, ino Ino, fh uint64, owner uint64, typ uint32, b
 	}
 	return
 }
+
+const (
+	FS_IOC_GETFLAGS = 0x80086601
+	FS_IOC_SETFLAGS = 0x40086602
+)
+
+func (v *VFS) Ioctl(ctx Context, ino Ino, cmd uint32, arg uint64, data []byte) (err syscall.Errno) {
+	if (cmd != FS_IOC_GETFLAGS && cmd != FS_IOC_SETFLAGS) || len(data) != 8 {
```

Comment:
The length of data is 4 in Ubuntu 20.04

Reply:
https://elixir.bootlin.com/linux/v5.19.2/source/include/uapi/linux/fs.h#L205